### DISCUSSION TITLE
Improving Documentation about variables for the cmake configuration

### DISCUSSION THREAD

**Discussion by bentxt** [c3lang/c3c#1389] [General]
Hi,
It would have been helpful to me if the cmake (?) had been mentioned in the "manual installation" section of the documentation. The specific variable was LLVM_DIR, which could be set like

```
export LLVM_DIR=/usr/local/opt/llvm@17
```

I think it would be a great addition to https://c3-lang.org/references/getting-started/setup/

**Comment by lerno** [comment]
Do you think you could make a pull request for it?
